seeds/xchange/xchange_trades_seed.csv

```diff
@@ -0,0 +1,4 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_bought_amount,token_sold_address,token_sold_amount
+ethereum,xchange,1,2023-06-12,0xf71d8fd364e2402eb11ba7bfbd9b59336e2b075571c1523d119f74a11a1986e3,170,0x117546D1467d80C6BdE13910412c724383260CF9,345579.7727468311529198,0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2,0.1
+ethereum,xchange,1,2023-06-12,0x7da422240ab88976df8c9694114daf8b8cfc335e1f851a8c175bca99e5e4cdcd,200,0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2,0.134508790612232095,0x117546D1467d80C6BdE13910412c724383260CF9,477750.8030874053723140
+ethereum,xchange,1,2023-06-12,0x4eba2bbed2a473a3a6b2fbfc1d784940b5157afc285c35aff1200e491ea1346a,115,0xDBc7945C5403c589Ec39A9Aa8C5AF234C706f6a2,14049529527.028758241707276441,0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2,0.1
```

Comment:
You can move/copy these records into the dex_trades.seed file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv